src/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -58,6 +58,7 @@ public partial class Process : Component
 
         private bool _raisedOnExited;
         private RegisteredWaitHandle _registeredWaitHandle;
+        private object _completionCallbackContext;
```

Comment:
This makes Process heavier weight on Windows even though it's only needed on Unix.